sources/packages/web/src/helpers/RouteHelper.ts

```diff
@@ -52,20 +52,22 @@ export class RouteHelper {
    */
   private static isAllowedIDP(
     clientType: ClientIdType,
-    idp: AppIDPType,
+    idp: IdentityProviders,
   ): boolean {
-    let allowedIDP = AppIDPType.UNKNOWN;
+    let allowedIDP: IdentityProviders | undefined = undefined;
```

Comment:
Sorry for my ignorance but what `undefined = undefined` means here?

Reply:
No need to be sorry 😉, it is no longer needed.
I changed to method to be more aligned with the one from API
![image](https://user-images.githubusercontent.com/61259237/215192865-11c13861-2c41-4098-b3f8-40145b41d486.png)